### PR TITLE
Added beginTextureFill with example and fixed mipmap generation

### PR DESCRIPTION
**Pull request by SmilyOrg**

### CODE REVIEW

**Comment by SmilyOrg** [comment]
Have to test on OSX and iOS

File: docs/examples/NanoVG/loom.config [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "sdk_version": "dev",
```
Comment (bengarney) [inline docs/examples/NanoVG/loom.config]: Should be latest.

**Comment by bengarney** [comment]
This looks good. Can you confirm it doesn't cause a perf regression on modest maps?

**Comment by SmilyOrg** [comment]
Confirmed fills working on all 4 plats. Perf seems more or less similar on modest maps. Maps on iOS exploded with errors, but that happened on master branch as well, so I'll have to look at that independently.

**Comment by bengarney** [comment]
I am ready to land this except for the "dev" example version, which needs to be corrected to "latest". Fix and I'll land!

**Comment by bengarney** [comment]
And thanks for the testing!

**Comment by SmilyOrg** [comment]
Forgot to PR #371 the second branch for that :)

**Pull request merged**